tests/pytorch_tests/model_tests/feature_models/gptq_test.py

```diff
@@ -201,3 +200,20 @@ def gptq_compare(self, ptq_model, gptq_model, input_x=None):
         # check all weights were not updated in gptq model compared to ptq model
         w_diffs = [np.isclose(np.max(np.abs(w_ptq - w_gptq)), 0) for w_ptq, w_gptq in zip(ptq_weights, gptq_weights)]
         self.unit_test.assertTrue(np.all(w_diffs), msg="GPTQ: some weights were updated in zero learning rate test")
+
+    def _extract_weights(self, model):
```

Comment:
seems like a useful helper method for tests, maybe put it in some common pytorch tests location so we'll be able to use it in the future?